compiler/src/iree/compiler/Dialect/Stream/Transforms/FoldUniformOperands.cpp

```diff
@@ -35,6 +36,54 @@ namespace {
 // Per-dispatchable export optimization
 //===----------------------------------------------------------------------===//
 
+// Returns the uniformly duplicated values across all indexing map entries of
+// duplicated values and a bit vector indicating the 'dead' values.
+//
+// Example:
+// Map of duplicated values per entry: [[0, 1, 0, 1], [0, 1, 0, 2]]
+// Returns:
+// - A map of uniformly duplicated values: [0, 1, 0, 2]
+// - A bit vector with dead values: b0010
```

Comment:
The implementation is `uniformDupeIndexMap[idx] = idx;`, so it is actually returning below:

```suggestion
// - A map of uniformly duplicated values: [0, 1, 0, 3]
// - A bit vector with dead values: b0010
```

Reply:
Thanks for catching, updated it!